### PR TITLE
docs: add third-party tools category with Kafka

### PR DESCRIPTION
**Pull request by bsmth**
__Additions:__
* Documentation page for Kafka Connect via JDBC driver.
* Top-level sidebar navigation category for Third-Party Tools

### CODE REVIEW

File: docs/third-party-tools/kafka.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+description: JDBC driver support in QuestDB allows for ingesting messages from a Kafka topic via Kafka Connect.
+---
+
+Support for the JDBC driver means that data can easily be exported from a Kafka
+topic and ingested directly to QuestDB by means of Kafka Connect.
+
+This article assumes that users have successfully set up an installation of
+Kafka using the Confluent platform. Detailed instructions on how to get up and
+running can be found on the
+[confluent.io documentation](https://docs.confluent.io/platform/current/quickstart/index.html).
```
Comment (mpsq) [inline docs/third-party-tools/kafka.md:12]:
```suggestion
[official documentation](https://docs.confluent.io/platform/current/quickstart/index.html).
```

File: docs/third-party-tools/kafka.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+## Configure PostgreSQL
+
+Create a new PostgreSQL database:
+
+```shell
+initdb -D "/path/to/db/pgdata"
+```
+
+Start PostgreSQL
```
Comment (mpsq) [inline docs/third-party-tools/kafka.md:33]:
```suggestion
Start PostgreSQL:
```

File: docs/third-party-tools/kafka.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+```shell
+pg_ctl -D "/path/to/db/pgdata" start
+```
+
+Connect to PostgreSQL
```
Comment (mpsq) [inline docs/third-party-tools/kafka.md:39]:
```suggestion
Connect to PostgreSQL:
```

File: docs/third-party-tools/kafka.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+auto.create=true
+```
+
+Start Kafka Connect:
+
+```shell
+bin/connect-standalone.sh config/connect-standalone.properties config/connect-jdbc.properties
+```
+
+## Publishing Messages
```
Comment (mpsq) [inline docs/third-party-tools/kafka.md:90]:
```suggestion
## Publish messages
```

We tend to avoid Pascal Case for titles.

**Review by mpsq — CHANGES_REQUESTED** [review]
Minor changes.

File: sidebars.js [hunk 5]
```diff
@@ -64,6 +64,13 @@ module.exports = {
         "develop/authenticate",
       ],
     },
+    {
+      label: "Third-Party Tools",
```
Comment (mpsq) [inline sidebars.js:68]:
```suggestion
      label: "Third-party tools",
```

**Review by mpsq — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 5 above)
Comment (bsmth) [reply sidebars.js:68]: 👍

**Review by mpsq — APPROVED** [review]
LGTM

**Pull request merged**